Felienne/hedy
# Address code review comments #1298

**Description**

- Fix translation of 'a number' in Dutch error messages
- Remove single quotes around in type error messages and changed them for code snippets
- Make the ask command similar to the print command in terms of allowed types

**Fix for**

#1298 

**How to test**

- At any level try `forward ten` and check that the suggested type is translated to Dutch properly. With the same example check that there are no single quotes and, instead, all code references are code snippets
- At level 12, check that the ask command accepts variables which are of type int or float: `a is 10\n answer is ask 'is your number ' number` should not yield a type error. NOTE that python will fail because changes are needed in the transpiler. To be addressed with #1351
- At level 16, check that the ask command accepts list variables: `colors is ['orange', 'blue', 'green']]\n favorite is ask 'Is your fav color' colors` should not yield a type error. Again python will fail because changes are needed in the transpiler.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
